compiler/one-cmds/onnx_legalizer.py

```diff
@@ -0,0 +1,749 @@
+#!/usr/bin/python3
+import onnx
+import onnx.numpy_helper
+import sys
+import numpy as np
+import re
+
+# Transform onnx model to make it compilable with our toolchain
+# List of transformations:
+# - Replace RNN operation with unrolled subgraph
+# - Replace LSTM operation with unrolled subgraph
+
+
+def reverse_str(s):
+    return ''.join(reversed(s))
```

Comment:
I think you already know about this. `s[::-1]` can do the same. But, what you implemented is more readable 👍

Reply:
actually, I did not know that it is applicable to strings =)